nilearn/interfaces/tests/test_bids.py

```diff
@@ -588,6 +590,8 @@ def test_save_glm_to_bids_contrast_definitions(
         "contrast-aaaMinusBbb_stat-t_statmap.nii.gz",
         "contrast-aaaMinusBbb_stat-variance_statmap.nii.gz",
         "contrast-aaaMinusBbb_stat-z_statmap.nii.gz",
+        "contrast-aaaMinusBbb_results.tsv",
+        "contrast-aaaMinusBbb_results.json",
```

Comment:
@tsalo I am running low on good idea on what suffix to use for saving tables of activations: any better idea than the very generic 'results'

Reply:
Maybe `clusters`? I think `results` is alright too though. @jdkent @JulioAPeraza might have some ideas, since I foresee the cluster table being used in NeuroVault at some point (as well as passed along to NiMARE, NeuroStore, etc.).

Reply:
I agree with @tsalo, "clusters" is a more specific (but still accurate) description of the results contained in the file (If I'm understanding that the file contains peak coordinates, the volume of the cluster, etc.)

Reply:
ok changing to clusters